h2o-k8s/README.md

```diff
@@ -0,0 +1,103 @@
+# H2O Kubernetes integration
```

Comment:
Would be good to provide utility to create docker image with H2O which can be used in Kuberentes pods and mention it in the docs, WDYT @Pscheidl?

If such image is released to official docker hub in each release of H2O, it would simplify using H2O a lot ( people won't need to build their images).

Plus Sparkling Water in external backend might just download the images and run the templates mentioned in the docs to start H2O on Kubernetes

Reply:
What we could do in SW is just to enhance these images with sparkling water extension jars and publish ours as well

Reply:
This is the plan. Adressing in a separate main thread, as this is important.

Reply:
Cool, makes sense to to this separately. Thanks!